### PR TITLE
[4.0] convert com_templates

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request für Issue #804 

### Zusammenfassung der Änderungen

convert com_templates up to https://github.com/joomlagerman/joomla/issues/963

### CODE REVIEW

File: administrator/language/de-DE/com_templates.ini [hunk 1]
```diff
@@ -180,19 +165,27 @@ COM_TEMPLATES_MSG_MANAGE_NO_STYLES="Es gibt <u>keine</u> installierten Stile zu
 COM_TEMPLATES_MSG_MANAGE_NO_TEMPLATES="Es gibt <u>keine</u> installierten Templates zu dieser Suchanfrage!"
 COM_TEMPLATES_N_ITEMS_DELETED="%d Template-Stile wurden gelöscht!"
 COM_TEMPLATES_N_ITEMS_DELETED_1="Der Template-Stil wurde gelöscht!"
+COM_TEMPLATES_N_OVERRIDE_CHECKED="%d Updated files list entry marked as checked."
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_templates.ini]: translation missing

File: administrator/language/de-DE/com_templates.ini [hunk 2]
```diff
@@ -180,19 +165,27 @@ COM_TEMPLATES_MSG_MANAGE_NO_STYLES="Es gibt <u>keine</u> installierten Stile zu
...
+COM_TEMPLATES_N_OVERRIDE_DELETED="%d Updated files list entry deleted."
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_templates.ini]: translation missing

File: administrator/language/de-DE/com_templates.ini [hunk 3]
```diff
@@ -180,19 +165,27 @@ COM_TEMPLATES_MSG_MANAGE_NO_STYLES="Es gibt <u>keine</u> installierten Stile zu
...
+COM_TEMPLATES_N_OVERRIDE_UNCHECKED="%d Updated files list entry marked as unchecked."
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_templates.ini]: translation missing

File: administrator/language/de-DE/com_templates.ini [hunk 4]
```diff
@@ -201,48 +194,43 @@ COM_TEMPLATES_PREVIEW="Vorschau"
...
-COM_TEMPLATES_TEMPLATE_MASTER_FILES="Template-Hauptdateien"
 COM_TEMPLATES_TEMPLATE_NEW_NAME_DESC="Den Namen des neuen Templates eingeben. Es dürfen nur Buchstaben, Zahlen und Unterstriche benutzt werden."
 COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL="Neuer Template-Name"
 COM_TEMPLATES_TEMPLATE_NO_PREVIEW="Keine Vorschau verfügbar! Die Vorschaufunktion kann in den Optionen aktiviert werden."
-COM_TEMPLATES_TEMPLATE_NO_PREVIEW_ADMIN="Keine Vorschau für das Administrationstemplate verfügbar!"
-COM_TEMPLATES_TEMPLATE_NO_PREVIEW_DESC="Um die Templatevorschau zu aktivieren, aktiviere die Vorschau Modulpositionen Option unter Templateoptionen."
 COM_TEMPLATES_TEMPLATE_NOT_SPECIFIED="Template nicht spezifiziert!"
 COM_TEMPLATES_TEMPLATE_PREVIEW="Vorschau"
 COM_TEMPLATES_TEMPLATES_FILTER_SEARCH_DESC="Im Templatenamen oder Verzeichnisnamen suchen."
+COM_TEMPLATES_TEMPLATES_TABLE_CAPTION="Table of Templates"
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_templates.ini]: translation missing

File: administrator/language/de-DE/com_templates.ini [hunk 5]
```diff
@@ -201,48 +194,43 @@ COM_TEMPLATES_PREVIEW="Vorschau"
...
 COM_TEMPLATES_TOGGLE_FULL_SCREEN="Für Vollbildbearbeitung Strg-Q drücken"
 COM_TEMPLATES_TOOLBAR_SET_HOME="Standard"
+COM_TEMPLATES_N_CONFLICT="%d Änderungen gefunden"
+COM_TEMPLATES_N_CONFLICT_1="Änderung gefunden"
+COM_TEMPLATES_UPTODATE="Auf dem neuesten Stand"
 COM_TEMPLATES_WARNING_FORMAT_WILL_NOT_BE_VISIBLE="Es wurde eine neue Datei mit der Endung „%s“ erstellt. Dieser Typ wird grundsätzlich unterstützt, wurde aber in den globalen Template-Optionen unter „Unterstützte Dateitypen“ nicht zur Anzeige freigegeben. Wenn dies erwünscht ist, bitte in der genannten Template-Option hinzufügen."
 COM_TEMPLATES_XML_DESCRIPTION="Diese Komponente verwaltet Templates"
+JLIB_HTML_PUBLISH_ITEM="Als überprüft markieren"
+JLIB_HTML_UNPUBLISH_ITEM="Als nicht-überprüft markieren"
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_templates.ini]:
```suggestion
JLIB_HTML_UNPUBLISH_ITEM="Als ungeprüft markieren"
```

File: administrator/language/de-DE/com_templates.ini [hunk 6]
```diff
@@ -201,48 +194,43 @@ COM_TEMPLATES_PREVIEW="Vorschau"
...
 COM_TEMPLATES_TEMPLATES_FILTER_SEARCH_DESC="Im Templatenamen oder Verzeichnisnamen suchen."
+COM_TEMPLATES_TEMPLATES_TABLE_CAPTION="Table of Templates"
 COM_TEMPLATES_TOGGLE_FULL_SCREEN="Für Vollbildbearbeitung Strg-Q drücken"
 COM_TEMPLATES_TOOLBAR_SET_HOME="Standard"
+COM_TEMPLATES_N_CONFLICT="%d Änderungen gefunden"
+COM_TEMPLATES_N_CONFLICT_1="Änderung gefunden"
+COM_TEMPLATES_UPTODATE="Auf dem neuesten Stand"
 COM_TEMPLATES_WARNING_FORMAT_WILL_NOT_BE_VISIBLE="Es wurde eine neue Datei mit der Endung „%s“ erstellt. Dieser Typ wird grundsätzlich unterstützt, wurde aber in den globalen Template-Optionen unter „Unterstützte Dateitypen“ nicht zur Anzeige freigegeben. Wenn dies erwünscht ist, bitte in der genannten Template-Option hinzufügen."
 COM_TEMPLATES_XML_DESCRIPTION="Diese Komponente verwaltet Templates"
+JLIB_HTML_PUBLISH_ITEM="Als überprüft markieren"
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_templates.ini]:
```suggestion
JLIB_HTML_PUBLISH_ITEM="Als geprüft markieren"
```

File: administrator/language/de-DE/com_templates.ini [hunk 7]
```diff
@@ -180,19 +165,27 @@ COM_TEMPLATES_MSG_MANAGE_NO_STYLES="Es gibt <u>keine</u> installierten Stile zu
...
 COM_TEMPLATES_NEW_FILE_HEADER="Eine neue Datei erstellen oder hochladen"
 COM_TEMPLATES_NEW_FILE_NAME="Neuer Dateiname"
 COM_TEMPLATES_NEW_FILE_SELECT="Dateityp auswählen"
 COM_TEMPLATES_NEW_FILE_TYPE="Dateityp"
 COM_TEMPLATES_NO_TEMPLATE_SELECTED="Kein Template ausgewählt"
 COM_TEMPLATES_OPTION_NONE=":: Keine ::"
-; Deprecated 4.0
-COM_TEMPLATES_OPTION_SELECT_PAGE="- Seite wählen -"
 COM_TEMPLATES_OPTION_SELECT_MENU_ITEM="- Menüeintrag wählen -"
+COM_TEMPLATES_OVERRIDE_CHECKED="Überprüft"
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_templates.ini]:
```suggestion
COM_TEMPLATES_OVERRIDE_CHECKED="Geprüft"
```

File: administrator/language/de-DE/com_templates.ini (hunk 1 above)
Comment (zero-24) [reply administrator/language/de-DE/com_templates.ini]:
```suggestion
COM_TEMPLATES_N_OVERRIDE_CHECKED="%d Änderungen wurden als geprüft markiert"
```

File: administrator/language/de-DE/com_templates.ini (hunk 3 above)
Comment (zero-24) [reply administrator/language/de-DE/com_templates.ini]:
```suggestion
COM_TEMPLATES_N_OVERRIDE_UNCHECKED="%d Änderungen wurden als ungeprüft markiert"
```

File: administrator/language/de-DE/com_templates.ini (hunk 2 above)
Comment (zero-24) [reply administrator/language/de-DE/com_templates.ini]:
```suggestion
COM_TEMPLATES_N_OVERRIDE_DELETED="%d Änderungen wurden gelöscht."
```

File: administrator/language/de-DE/com_templates.ini (hunk 4 above)
Comment (zero-24) [reply administrator/language/de-DE/com_templates.ini]:
```suggestion
COM_TEMPLATES_TEMPLATES_TABLE_CAPTION="Tabelle der Templates"
```

**Comment by zero-24** [comment]
Done the missing translations

File: administrator/language/de-DE/com_templates.ini [hunk 8]
```diff
@@ -180,19 +165,27 @@ COM_TEMPLATES_MSG_MANAGE_NO_STYLES="Es gibt <u>keine</u> installierten Stile zu
 COM_TEMPLATES_MSG_MANAGE_NO_TEMPLATES="Es gibt <u>keine</u> installierten Templates zu dieser Suchanfrage!"
 COM_TEMPLATES_N_ITEMS_DELETED="%d Template-Stile wurden gelöscht!"
 COM_TEMPLATES_N_ITEMS_DELETED_1="Der Template-Stil wurde gelöscht!"
+COM_TEMPLATES_N_OVERRIDE_CHECKED="%d Änderungen wurden als geprüft markiert"
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_templates.ini]:
```suggestion
COM_TEMPLATES_N_OVERRIDE_CHECKED="%d Änderungen wurden als geprüft markiert."
```

File: administrator/language/de-DE/com_templates.ini [hunk 9]
```diff
@@ -180,19 +165,27 @@ COM_TEMPLATES_MSG_MANAGE_NO_STYLES="Es gibt <u>keine</u> installierten Stile zu
...
+COM_TEMPLATES_N_OVERRIDE_DELETED="%d Änderungen wurden gelöscht."
+COM_TEMPLATES_N_OVERRIDE_UNCHECKED="%d Änderungen wurden als ungeprüft markiert"
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_templates.ini]:
```suggestion
COM_TEMPLATES_N_OVERRIDE_UNCHECKED="%d Änderungen wurden als ungeprüft markiert."
```

**Pull request merged**